### PR TITLE
docs: add exercises for intermediate/state

### PR DESCRIPTION
**Pull request by ericbolikowski**
Heya you all 👋 ,

## What is this PR about?
I added the codesandboxed exercises for level: intermediate > topic: state therefore the README.md file has been updated.

Please do some QA and review the following branches and codesandboxes before approving this PR:
-  *kodiri/intermediate/state/01* [State 01: Display a message from the component state](https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/kodiri/intermediate/state/01)
- *kodiri/intermediate/state/02* [State 02: Display the average score of a player](https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/kodiri/intermediate/state/02)
- *kodiri/intermediate/state/03* [State 03: Display the average score of a player after completing a challenge](https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/kodiri/intermediate/state/03)
- *kodiri/intermediate/state/04* [State 04: Perform some math operations](https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/kodiri/intermediate/state/04)
- *kodiri/intermediate/state/05* [State 05: Add new element to state](https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/kodiri/intermediate/state/05)

Happy reviewing! 🌹 🕊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,19 +46,19 @@ Click the below links to load a specific branch from this repository into codesa
 
 - [Input 01: Display a basic input text](https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/cp/kodiri/beginner/input/01)
 - [Input 02: Display an input text with dynamic data](https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/cp/kodiri/beginner/input/02)
-- [Input 03: Display a textarea with dynamic data](https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/cp/kodiri/beginner/input/03)
+- [Input 03: Display a textarea wi<!-- t -->h dynamic data](https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/cp/kodiri/beginner/input/03)
```
Comment (paschalidi) [inline README.md]: what is this achieving?

**Review by paschalidi — COMMENTED** [review]
Looks great thanks for the work @ericbolikowski 

Few minor comments. 
1. https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/kodiri/intermediate/state/03 has no props given to the component on the `index.js`
2. Same here I believe https://codesandbox.io/s/github/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/tree/kodiri/intermediate/state/04

And a minor comment that doesnt need to be fixed but would be maybe cool to:
https://github.com/ReDI-School/react-berlin-2019-fall-codesandbox-exercises/blob/kodiri/intermediate/state/01/src/index.js#L15 Here there is a small message I implemented for the students who hover over the 🔫, you can think if you want to update it ;)

**Review by sergioh213 — APPROVED** [review]
After responding to @paschalidi 's comment, I'm good with the PR

File: README.md (hunk 1 above)
Comment (ericbolikowski) [reply README.md]: Just some clumsy typing in vscode. Fixed.

**Comment by ericbolikowski** [comment]
Great, thank you @paschalidi + @sergioh213. I've fixed all issues. Updates exercises 01-04 with some quotes from here :) https://www.devglan.com/blog/best-developer-quotes

**Comment by paschalidi** [comment]
@ericbolikowski you genius man thanks!

**Pull request merged**